docs/index.txt

```diff
@@ -45,13 +45,25 @@ Each user should also :ref:`install the Determined command-line tools <install-c
 
 Next Steps
 """"""""""
+
 To get started using Determined, check out the :ref:`quick-start`.
 
-Next, follow the tutorial for your preferred deep learning framework:
+Determined has two Experiment APIs: Trial and Native.  With the :ref:`model-definitions_trial-api`, developers expose the components of a model definition to Determined’s training loop implementation.  With the :ref:`model-definitions_native-api` (TensorFlow Keras and Estimators), developers instrument training scripts adhering to these frameworks' native patterns including their training loops.
+
+If you're using TensorFlow Keras or Estimator and want to get started quickly
+with a minimal modification of source code, follow our Native API tutorial:
+
+- :ref:`tutorials_native-api`
+
+If you're using PyTorch or want a finer-grained level of control over the
+training loop, follow our Trial API tutorials in your preferred framework:
 
 - :ref:`pytorch-mnist-tutorial`
 - :ref:`tf-mnist-tutorial`
 
+The command line interface and Python SDK support the Trial API.  The Python
```

Comment:
blocking: As above, describe the two APIs in the same order every time they're paired here.